app/models/goal.py

```diff
@@ -1,5 +1,19 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import String
 from ..db import db
 
 class Goal(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str] = mapped_column(String(50))
```

Comment:
Consider whether the this column for `Goal` should be nullable. It feels odd to allow someone to create a goal that gets saved to the DB without a title.